0005-snarkyjs-ecdsa-sha.md

```diff
@@ -148,31 +148,158 @@ Overall, exposing new gadgets and gates follow a strict pattern that has been us
 
 ## ECDSA
 
-**TODO** this will be a seperate PR stacked on top of this one
+Similar to SHA3 and Keccak, the gadget for ECDSA has been implemented by the crypto team and is available through exposing it in the bindings layer. However, designing and implementing a safe API for ECDSA is not as straight forward as it is for SHA3 and Keccak.
 
-**Evergreen, wide-sweeping Details**
+The main verification step of ECDSA is implemented via a `verify` gadget in OCaml.
 
-**Ephemeral details**
+```ocaml
```

Comment:
All this information about the OCaml interface is great but should be in a comment in the OCaml implementation and linked to from this doc!